mautic/mautic
# [backport] Empty the (un)publish dates on email clone

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ x ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

When cloning an email, the publish up and down dates are copied over. This can cause issue when clients do not notice and publish the email but have no intention of automatically sending it (yet) using broadcast:send.

This PR simply ensures those fields are empty on clone.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a Segment email
3. Give it a published up and down date
4. Clone that email
5. Check that those dates are gone in the clone
